Extensions/Physics2Behavior/physics2runtimebehavior.ts

```diff
@@ -719,10 +719,6 @@ namespace gdjs {
         this.createBody();
       }
 
-      // Reset contacts that happened this frame
-      this.contactsStartedThisFrame.length = 0;
-      this.contactsEndedThisFrame.length = 0;
-
       // Step the world if not done this frame yet
       if (!this._sharedData.stepped) {
         this._sharedData.step(
```

Comment:
I think it should moved inside `step(deltaTime)`.

Reply:
I was not detected by tests probably because the checked object was the 1st to run. The test could create an object before and after the ones that are used to cover the case.

Reply:
I guess so, but I'd rather postpone this for another PR in order to fix the current bug

Reply:
It's not like it would take a lot of time.
Without this change the condition is still broken for extensions.

Reply:
I would rather do ths larger change in another PR, because here we want to fix asap the version that is in the wild, and we can't take the risk of introducing another "subtle" behavior change that will get us deeper into the rabbit hole of bugs.